airflow/models/serialized_dag.py

```diff
@@ -0,0 +1,173 @@
+# -*- coding: utf-8 -*-
+#
+# Licensed to the Apache Software Foundation (ASF) under one
+# or more contributor license agreements.  See the NOTICE file
+# distributed with this work for additional information
+# regarding copyright ownership.  The ASF licenses this file
+# to you under the Apache License, Version 2.0 (the
+# "License"); you may not use this file except in compliance
+# with the License.  You may obtain a copy of the License at
+#
+#   http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+
+"""Serialzed DAG table in database."""
+
+import hashlib
+from datetime import timedelta
+from typing import Any, Dict, List, Optional, TYPE_CHECKING
+
+from sqlalchemy import Column, Index, Integer, String, Text, and_
+from sqlalchemy.sql import exists
+
+from airflow.models.base import Base, ID_LEN
+from airflow.utils import db, timezone
+from airflow.utils.log.logging_mixin import LoggingMixin
+from airflow.utils.sqlalchemy import UtcDateTime
+
+if TYPE_CHECKING:
+    from airflow.dag.serialization.serialized_dag import SerializedDAG  # noqa: F401, E501; # pylint: disable=cyclic-import
+    from airflow.models import DAG  # noqa: F401; # pylint: disable=cyclic-import
+
+log = LoggingMixin().log
+
+
+class SerializedDagModel(Base):
```

Comment:
```suggestion
class SerializedDag(Base):
```

And then call this `SerialzedDagModel` at import time if we need to distinguish this.

Reply:
The problem is it will collide a lot in the same file & so become confusing:

https://github.com/apache/airflow/blob/e2570ca0002394af9f59703a2b29ad21642c546b/airflow/models/serialized_dag.py#L73

https://github.com/apache/airflow/blob/e2570ca0002394af9f59703a2b29ad21642c546b/airflow/models/serialized_dag.py#L34-L37`

I can obviously add `as` on those imports to change but not sure if it is worth it, WDYT?